blueos_repository/consolidate.py

```diff
@@ -46,6 +54,13 @@ def from_json(json_dict: Dict[str, str]) -> "Company":
         return Company(name=json_dict["name"], email=json_dict.get("email", None), about=json_dict.get("about", None))
 
 
+class ExtensionType(StrEnum):
+    DEVICE_INTEGRATION = "device-integration"
+    EXAMPLE = "example"
+    THEME = "theme"
+    OTHER = "other"
```

Comment:
```python
class ExtensionType(HumanStringEnum):
    DEVICE_INTEGRATION = auto()
    EXAMPLE = auto()
    THEME = auto()
    OTHER = auto()
```

Reply:
This doesn't create desirable values.

Reply:
Yes, it prints nice things, but the actual values end up as string versions of numbers.
```python
>>> ExtensionType.DEVICE_INTEGRATION
<ExtensionType.DEVICE_INTEGRATION: '1'>
>>> ExtensionType.DEVICE_INTEGRATION == 'device-integration'
False
>>> ExtensionType.DEVICE_INTEGRATION == '1'
True
```